advisories/hackage/git-annex/HSEC-2023-0013.md

```diff
@@ -20,6 +20,7 @@ type = "ARTICLE"
 url = "https://git-annex.branchable.com/upgrades/insecure_embedded_creds/"
 ```
 
+
```

Comment:
unrelated?

Reply:
Yes, that was to force the action, but it was failing for another reason. The tool presently doesn't' accept the schema:

```
strict mode: missing type "array" for keyword "contains" at "https://raw.githubusercontent.com/ossf/osv-schema/main/validation/schema.json#/properties/affected/items/properties/ranges/items/allOf/1/if/properties/events" (strictTypes)
strict mode: missing type "object" for keyword "required" at "https://raw.githubusercontent.com/ossf/osv-schema/main/validation/schema.json#/properties/affected/items/properties/ranges/items/allOf/1/if/properties/events/contains" (strictTypes)
```

I'll look into using another validator.